types/markdown-to-jsx/index.d.ts

```diff
@@ -82,6 +82,12 @@ export interface MarkdownOptions {
 
     /** Custom function to generate an HTML id from headings. */
     slugify?: (text: string) => string;
+
+    /** List of html codes are converted to unicode characters */
```

Comment:
Can you update the markdown-to-jsx version number this applies to at the top of the file? Otherwise looks good.

Reply:
The version is correct, I think the one we use is even lower.